### ISSUE TITLE
Adding labels to project to start adding them to the brainhack jupyter book database

### ISSUE THREAD

**Issue by Remi-Gau** [ohbm/hackathon2023#40]
@anibalsolon 

I will start adding labels to the project issues to then be able to easily grab them with the github API and them to our "big csv of all projects" (BCOAP).

https://github.com/brainhackorg/brainhack_jupyter_book/blob/main/data/hackathon_projects.tsv

I suspect I should not remove the `Project` and `Good To Go` labels.

Anything else before I start adding labels all over the place?

**Comment by anibalsolon** [comment]
Right, I'd recommend getting only issues with these two tags, given we will use the issues for the proceedings as well e.g. https://github.com/ohbm/hackathon2023/issues/39
